src/Pirouette/Transformations/Tagged.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE StandaloneKindSignatures #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE TypeFamilyDependencies #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ConstraintKinds #-}
+
+module Pirouette.Transformations.Tagged where
+
+import Data.Kind
+import GHC.TypeLits
+
+data Foo
+data Bar
+data Baz
+
```

Comment:
I believe we no longer need these toy types.

Reply:
Oops, I had a feeling I forgot to remove something, and here it is!